scripts/setup_test_lab.sh

```diff
@@ -48,37 +48,37 @@ source common.sh
 print_usage() {
     echo "Usage:"
     echo "  cd scripts"
-    echo "  export KEY_NAME=<your ec2 key name>"
-    echo "  export KEY_FILE=<path-to-ec2-key-file>"
     echo "  export NAME_TAG=<name_tag to identify your resources>"
     echo "  and then run ./setup-test-lab.sh [optional: si definition file name, yes, name without path, 'scripts/si_def.json' if not provided"
 }
 
-if [ -z ${NAME_TAG:=} ] || [ -z ${KEY_NAME:=} ] || [ -z ${KEY_FILE:=} ]
+if [ -z ${NAME_TAG:=} ]
```

Comment:
I think one of the reason KEY_NAME and KEY_FILE were originally set as env var is so that  people can use their own keys without having to change any checked in file.

Reply:
this argument makes sense. so I will drop this PR.